shell/app/styles/antd-extension.scss

```diff
@@ -336,3 +336,15 @@ $border-style: 1px solid $color-border;
     height: 100%;
   }
 }
+
+sup.ant-badge-dot {
```

Comment:
Add some comment for this otherwise we don't know why is it exist.

Reply:
added comments